GoogleSignIn.podspec

```diff
@@ -46,7 +46,7 @@ The Google Sign-In SDK allows users to sign in with their Google account from th
   s.test_spec 'unit' do |unit_tests|
     unit_tests.platforms = {
       :ios => ios_deployment_target,
-      :osx => osx_deployment_target
+      :osx => '11.0'
```

Comment:
We can probably revert this change as the linker warning being generated is only given "NOTE" status (rather than "WARN") in the build process and does not fail the build.

Reply:
Done.